server/src/controllers/auth/auth.js

```diff
@@ -15,7 +15,7 @@ exports.login = (req, res) => {
   console.log('--------------------------------------------------');
   console.log(req.body);
```

Comment:
Please don't do this.  Use `.env` and change the `LOG_LEVEL` to debug you will see the whole request and response payload on your console.  I won't accept this kind of changes on our main repo.

Reply:
I remove this console.log in the next commit